clients/java/README.md

```diff
@@ -22,19 +22,17 @@ implementation 'io.github.marquezproject:marquez-java:0.19.1
 
 ## Usage
 
+### Metadata Collection
+Marquez implements OpenLineage specification. Use [OpenLineage client](https://github.com/OpenLineage/OpenLineage/tree/main/client/java) to collect metadata.
```

Comment:
_Minor_: I'd phrase this as:

> To collect [OpenLineage](https://openlineage.io) events using Marquez, please use the [`openlineage-java`](https://search.maven.org/artifact/io.openlineage/openlineage-java) library. OpenLineage is an _Open Standard_ for lineage metadata collection designed to collect metadata for a job in execution.

Reply:
I'd also break up the sections as follows:

### Reading Metadata

### Writing Metadata

Reply:
Thank you for the feedback. Let me rephrase it.